cumulusci/tasks/bulkdata/extract.py

```diff
@@ -97,8 +97,9 @@ def _init_db(self):
         with self._database_url() as database_url:
 
             # initialize the DB engine
-            self.engine = create_engine(database_url)
-            with self.engine.connect() as connection:
+            parent_engine = create_engine(database_url)
+            with parent_engine.connect() as connection:
+                self.engine = connection
```

Comment:
This is now storing a connection, not an engine. Can we update the code to use self.connection instead of self.engine, for clarity?

(If that is a tricky change to make for some reason, I think we could keep this the way it was, but be sure to explicitly close the session and dispose the engine in a finally block, like some of the tests were doing.)